stubs/livewire/resources/views/components/action-message.blade.php

```diff
@@ -1,10 +1,14 @@
 @props(['on'])
 
 <div x-data="{ shown: false, timeout: null }"
-    x-init="@this.on('{{ $on }}', () => { clearTimeout(timeout); shown = true; timeout = setTimeout(() => { shown = false }, 2000);  })"
-    x-show.transition.out.opacity.duration.1500ms="shown"
-    x-transition:leave.opacity.duration.1500ms
-    style="display: none;"
+     x-on:{{ $on }}.window="
```

Comment:
Before the even listener was scoped to the current component, this one appears not to be. (like we paired on @nunomaduro )

Reply:
There is a actually a way of scoping this listener to the current component? I was not clear to me - that L3 provides an alternative the current API.